integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoTestNonTransactionalTableFiles.scala

```diff
@@ -230,6 +230,37 @@ class PrestoTestNonTransactionalTableFiles extends FunSuiteLike with BeforeAndAf
     }
   }
 
+  def buildOnlyBinary(rows: Int, sortColumns: Array[String], path : String): Any = {
```

Comment:
existing binary testcase only can you add filter query ? I guess no need to add new testcase for it, it will slow down CI running time.

Reply:
done